targets/wioterminal.json

```diff
@@ -0,0 +1,8 @@
+{
+    "inherits": ["atsamd51p19a"],
+    "build-tags": ["sam", "atsamd51p19a", "wioterminal"],
+    "flash-1200-bps-reset": "true",
+    "flash-method": "msd",
+    "msd-volume-name": "Arduino",
```

Comment:
Just verifying that this is the correct volume name here?

Reply:
Yes, that's the correct volume name.
I can flash tinygo with this setting.
![image](https://user-images.githubusercontent.com/9251039/83886908-a986eb00-a782-11ea-99dd-10b28f67fe5c.png)